app/views/spree/api/line_items/bulk_show.v1.rabl

```diff
@@ -1,5 +1,7 @@
 object @line_item
-attributes :id, :quantity, :max_quantity
+attributes :id, :quantity, :max_quantity, :price
```

Comment:
I don't think we need :price here when we have a node named :price below? I could be wrong